torch/distributed/__init__.py

```diff
@@ -56,33 +56,35 @@ def is_available() -> bool:
         _make_nccl_premul_sum,
     )
 
+    class DistributedPdb(pdb.Pdb):
```

Comment:
do we want to also make this a public API or we want `breakpoint` be the single public API?

Reply:
not sure, i left it as public thinking that someone might want to further customize. but it is not critical, and we could just make it private for now, which is probably a defensive choice